README.md

```diff
@@ -36,12 +36,6 @@ Repo: [github.com/NebraLtd/hm-gatewayrs](https://github.com/NebraLtd/hm-gatewayr
 
 This container is the actual Helium Gateway-rs software (from upstream), with the required configuration files added.
 
-## DBus Session
```

Comment:
On line seven:

> There are currently six different services running within this device, which are all outlined below.

Change this to four now

Reply:
will. Missed.

Reply:
done